vector/src/main/java/im/vector/app/core/utils/Dialogs.kt

```diff
@@ -40,16 +45,36 @@ fun Context.displayInWebView(url: String) {
             .show()
 }
 
-fun Context.showIdentityServerConsentDialog(configuredIdentityServer: String?, policyLinkCallback: () -> Unit, consentCallBack: (() -> Unit)) {
+fun Context.showIdentityServerConsentDialog(identityServerWithTerms: IdentityServerWithTerms?,
```

Comment:
general question about identity servers, are privacy policies enforced?

Reply:
You mean can we have an IS without any privacy policy? I do not think so, but the code should be robust to that.
By the way I will create some issues (defect, but no P1) I have seen in the area